### PR TITLE
refactor(config-ui): the blueprint setting page

### PR DESCRIPTION
**Pull request by mintsweet**
### Summary
refactor the **blueprint setting page**.

### Screenshots
screenshot-20221215-095210

### CODE REVIEW

File: config-ui/src/pages/blueprint/detail/blueprint-detail.tsx [hunk 1]
```diff
@@ -20,21 +20,25 @@ import React, { useState } from 'react'
...
 
 export const BlueprintDetail = ({ id }: Props) => {
   const [activeTab, setActiveTab] = useState<TabId>('configuration')
 
-  const { loading, blueprint, saving, onUpdate } = useDetail({ id })
+  const { loading, blueprint, connections, saving, onUpdate } = useDetail({
+    id
+  })
+
+  console.log(connections)
```
Comment (likyh) [inline config-ui/src/pages/blueprint/detail/blueprint-detail.tsx:41]: delete debug code

File: config-ui/src/pages/blueprint/detail/panel/configuration.tsx [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+import { Table, ColumnType } from '@/components'
+import { getCron } from '@/config'
+import { DataScopeList } from '@/plugins'
+
+import type { BlueprintType, ConnectionItemType } from '../types'
+import {
+  UpdateNameDialog,
+  UpdatePolicyDialog,
+  UpdateScopeDialog,
```
Comment (likyh) [inline config-ui/src/pages/blueprint/detail/panel/configuration.tsx:31]: Can the existing scope be selected as default in UpdateScopeDialog?

File: config-ui/src/pages/blueprint/detail/blueprint-detail.tsx (hunk 1 above)
Comment (mintsweet) [reply config-ui/src/pages/blueprint/detail/blueprint-detail.tsx]: done.

File: config-ui/src/pages/blueprint/detail/panel/configuration.tsx (hunk 2 above)
Comment (mintsweet) [reply config-ui/src/pages/blueprint/detail/panel/configuration.tsx:31]: It's not supported yet, I'm thinking about it.

**Review by likyh — APPROVED** [review]
Some bugs left will fix on the bug fix period.

**Pull request merged**